compiler/p/runtime/PPCArrayCopy.inc

```diff
@@ -2437,11 +2435,6 @@ FUNC_LABEL(__forwardLEArrayCopy):
 __forwardLEArrayCopy:
 #endif
 startproc.__forwardLEArrayCopy:
-
-   cmpli cr0, CmpAddr, r7, 17
```

Comment:
Can you please clarify why the implementations of `forwardLEArrayCopy` and `leArrayCopy` are being removed as part of this change?

At the very least, these removals should probably be in a separate commit.

Reply:
Answer above.